README.md

```diff
@@ -0,0 +1,52 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+<p align="center">Desktop view</p>
+<img src="./images/web.jpg">
+
+<p align="center"> Mobile view</p>
+<img src="./images/mobile.jpg"> 
+
+<h1 align="center"> Module One Capstone Project </h1>
+
+> This is a repository for my Module One Capstone Project at Microverse. It is a website for a fictitious, upcoming F1 Sustainability Summit.
+> Visit the [Live Link](https://benmuiruri.github.io/f1-sustainability-capstone/) to view the website.
+## Built With
```

Comment:
- [ ] OPTIONAL:  The live link is broken, it opens an empty page, please make sure that the link works as expected

Reply:
The live link is empty because I am yet to merge. I had not set it to render from this branch. Once I merge the page will render as expected.